### ISSUE TITLE
Various Problems with EditableColumn

### ISSUE THREAD

**Issue by chodges** [kartik-v/yii2-grid#486]
First off Kartik I just want to say that your controls are **indispensable** and I really really appreciate your work.  

I'm also having a bunch of problems with this too.  i'm noticing that if I have only one editable column then the value NEVER gets sent to the control when I press the Pop-Ups submit button.   The 2nd editable column does send the value. 

BUT what I've also found is that if I use a the INPUT_SPIN it is not populating the spinner with my min/max range.  

So there seems to be a lot of issues with this control.

```
    [
        'class' => 'kartik\grid\EditableColumn',
        'attribute'=>'discount',
        'header'=>'Discount',
        'editableOptions'=>[
            'header'=>'Discount',
            'inputType'=>\kartik\editable\Editable::INPUT_SPIN,
            'options'=>['pluginOptions'=>['min'=>0, 'max'=>5000]]
        ],
        'hAlign'=>'right',
        'width'=>'100px',
        'format'=>['decimal', 2],
    ],

    [
        'class' => 'kartik\grid\EditableColumn',
        'attribute'=>'discount_override',
        'header'=>'Discount Override',
        'editableOptions'=>[
            'header'=>'Discount Override',
            'inputType'=>\kartik\editable\Editable::INPUT_SPIN,
            'options'=>['pluginOptions'=>['min'=>0, 'max'=>5000]]
        ],
        'hAlign'=>'right',
        'width'=>'100px',
        'format'=>['decimal', 2],
    ],
```

//And the Model:

```
    class OpportunityProduct extends \yii\db\ActiveRecord
    {
        /**
         * @inheritdoc
         */
        public static function tableName()
        {
            return 'opportunity_product';
        }

        /**
         * @inheritdoc
         */
        public function rules()
        {
            return [
                [['opportunity_id', 'product_id'], 'required'],
                [['opportunity_id', 'product_id'], 'integer'],
                [['base_fee'], 'number'],
                [['discount', 'discount_override'], 'number', 'min'=>0, 'max'=>100],
                [['opportunity_id'], 'exist', 'skipOnError' => true, 'targetClass' => Opportunity::className(), 'targetAttribute' => ['opportunity_id' => 'opportunity_id']],
                [['product_id'], 'exist', 'skipOnError' => true, 'targetClass' => Product::className(), 'targetAttribute' => ['product_id' => 'product_id']],
            ];
        }
```

When I try to edit the value look at my Spinner (First of all the pop-up is WAY too small.  It looks nothing like the layout of the pop-ups in Kartik's demo page.  But more importantly even though I'm specifying the min/max range **the values are not showing up in the spinner**:

<img width="329" alt="screen shot 2016-05-20 at 9 56 37 pm" src="https://cloud.githubusercontent.com/assets/197811/15446435/c1ac8468-1ed5-11e6-84a3-a7955bbed534.png">

**And here is one of the errors I'm seeing when I submit the change:**

<img width="970" alt="screen shot 2016-05-20 at 10 00 24 pm" src="https://cloud.githubusercontent.com/assets/197811/15446459/524d7342-1ed6-11e6-86e7-ac3fd29e8c67.png">

So for right now the editable column is useless to me right now and I'm going to have to use another solution to make this work.  Any input you can provide Kartik would be soooo helpful.

Thank you.

**Comment by kartik-v** [comment]
Unfortunately cannot reproduce your issue.

Can you [cross check the yii2-grid demo page](http://demos.krajee.com/grid-demo) and confirm? An editable column similar to your use case is been set for **Buy Amount** and cannot see any of the issues you mention.

**Comment by jonmixnblend** [comment]
Hi @chodges , I had a similar issue with that html.replace is not a function.

In my case it was caused by

```
require(__DIR__ . '/../config/bootstrap.php');
```

in my index.php file, using the advanced app, where that file just had:

```
<?php
```
in it and nothing else. This meant that <?php was getting output in any json response. I'd say check your json response, and check that it is all good. Once the json response was fine, then that issue went away.

As far as the popup not looking anything like on the demo page, are you using any sort of Bootstrap theme, that maybe includes styling for the editable popup etc in it?

Hope what I've posted here helps you.
